modules/autofill/autofill-api/src/main/java/com/duckduckgo/autofill/InternalTestUserChecker.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill
+
+/**
+ * Public API that could be used if the user is a verified internal tester.
+ * This class could potentially be moved to a different module once there's a need for it in the future.
+ */
+interface InternalTestUserChecker {
```

Comment:
NIT: i think this can leave somewhere else at some point as it is not entirely Autofill specific. Do you have a suggestion where?

We can also do it in the future when it will make more sense.

Reply:
>We can also do it in the future when it will make more sense.

Yeah, I think it's fine here for now and when a second use case arises (if it does), it'll be easier to re-home it with that context in mind.